docs/source/common/lightning_module.rst

```diff
@@ -841,7 +841,7 @@ The current step (does not reset each epoch)
 
 hparams
 ~~~~~~~
-After calling `save_hyperparameters` anything passed to init() is available via hparams.
+After calling ``save_hyperparameters`` anything passed to ``__init__()`` is available via the ``hparams`` attribute.
```

Comment:
It's not anything passed, right? @rohitgr7 

```suggestion
The arguments saved by calling ``save_hyperparameters`` passed through ``__init__()`` could be accessed by the ``hparams`` attribute.
```

Reply:
:+1: ... can you update this in your PR: https://github.com/PyTorchLightning/pytorch-lightning/pull/6056??

Reply:
@Borda Updated it here https://github.com/PyTorchLightning/pytorch-lightning/pull/6065 :)